net/api/impl/apple_client.h

```diff
@@ -91,7 +91,7 @@ struct ImplWrapper<HTTPClientApple> {
                                  HTTPResponse& output) {
     // TODO(dkorolev): Handle redirects in Apple implementation.
     output.url = response.url_received;
-    output.code = response.error_code;
+    output.code = static_cast<HTTPResponseCodeValue>(response.error_code);
```

Comment:
1. :+1: 
2. I believe there's a constructor in `HTTPResponseCode` that takes an `int`. Thus, `HTTPResponseCode(response.error_code)` would do the trick in a slightly more readable fashion.
3. Perhaps we should rename `error_code` into some `http_response_code`?